ocs_ci/deployment/deployment.py

```diff
@@ -4049,3 +4048,38 @@ def deploy_dr_policy(self):
     "regional-dr": RDRMultiClusterDROperatorsDeploy,
     "metro-dr": MDRMultiClusterDROperatorsDeploy,
 }
+
+
+def get_arbiter_location():
```

Comment:
In my WIP branch for 2AZ+Arbiter I've moved this to it's own utility module `utility/arbiter.py` to try to avoid any circular imports by leaving it in the deployment module. Not sure if you've hit these but if you are making any other changes to the branch perhaps this could be moved as well.